swig/cpp/src/Internal.hpp

```diff
@@ -86,63 +85,54 @@
 #define S_Restr                  std::shared_ptr<Restr>
 #define S_Ident                  std::shared_ptr<Ident>
 
-#define NEW(data, element, class)                                                                                                                    \
+#define LY_NEW(data, element, class)                                                                                                                 \
```

Comment:
Please consider *not* placing these backslashes so much to the right -- I just lost a considerable amount of time figuring out how the hell this can actually work given that "there are no backslashes" :).

Reply:
Sorry, this was automatically formatted.

Do you have a tool or a style guide for this?

I'm using Clang format with the code style "LLVM".